webapp/views/launch/script.js

```diff
@@ -142,6 +142,25 @@ function handleResponse(res) {
           break;
 
         // == time map module ==
+        // Travel mode
+        case 'time_map.12':
+          form = formElement(messageId);
+          lists.append($(`<select id="${messageId}-input" class='custom-select' size="10">` + ['Automobile', 'Bicycle', 'Walk'].map(col => `<option selected value="${col}">${col}</option>`) + `</select>`));
```

Comment:
I don't like the appaerance of the select element ... as there are only three options, we could use nice, large buttons.